src/lib/core/a11y/list-key-manager.ts

```diff
@@ -63,6 +64,9 @@ export class ListKeyManager<T extends CanDisable> {
       case END:
         this.setLastItemActive();
         break;
+      case ESCAPE:
```

Comment:
Isn't a service for this overkill? The only thing it would do is to expose `Observable.fromEvent(document, keydown).filter(e => e.keyCode === ESCAPE)`. I think that at this point it would be better to just inline it or bake it into the overlay directive.